src/redux/actions.js

```diff
@@ -1,64 +1,32 @@
-import { types } from './types';
-import axios from "axios";
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { actionTitles } from './actionTitles';
 
-export const setContacts = array => ({
-  type: types.SET_CONTACT,
-  payload: array,
-});
+const MOCKAPI_URL = 'https://61546f8d2473940017efae5d.mockapi.io';
 
-export const addContact = data => ({
-  type: types.ADD_CONTACT,
-  payload: data,
-});
-
-export const deleteContact = id => ({
-  type: types.DELETE_CONTACT,
-  payload: id,
-});
-
-export const contactFetchStart = () => ({
-  type: types.CONTACT_FETCH_START,
-});
-
-export const contactFetchFinished = () => ({
-  type: types.CONTACT_FETCH_FINISHED,
-});
-
-export const contactFetchError = (error) => ({
-  type: types.CONTACT_FETCH_ERROR,
-  payload: error,
-});
-
-export const getContactsOperation = () => async (dispatch) => {
-  dispatch(contactFetchStart());
-  try {
-    const result = await axios.get("https://61546f8d2473940017efae5d.mockapi.io/contacts");
-    dispatch(setContacts(result.data));
-  } catch (error) {
-    dispatch(contactFetchError(error));
-  } finally {
-    dispatch(contactFetchFinished());
+export const createContact = createAsyncThunk(
+  actionTitles.createContact,
+  async (contact) => {
+    const result = await axios.post(`${MOCKAPI_URL}/contacts`, contact);
+    return result.data;
   }
-};
+);
 
-export const postContactOperation = (contact) => async (dispatch) => {
-  dispatch(contactFetchStart());
-  try {
-    const result = await axios.post("https://61546f8d2473940017efae5d.mockapi.io/contacts", contact);
-    dispatch(addContact(result.data));
-  } catch (error) {
-    dispatch(contactFetchError(error));
-  } finally {
-    dispatch(contactFetchFinished());
+export const getContacts = createAsyncThunk(
+  actionTitles.getContacts,
+  async () => {
+    const result = await axios.get(`${MOCKAPI_URL}/contacts`);
+    return result.data;
   }
-};
-
-export const deleteContactOperation = (id) => async (dispatch) => {
-  try {
-    const result = await axios.delete(`https://61546f8d2473940017efae5d.mockapi.io/contacts/${id}`);
-    dispatch(deleteContact(id));
-    console.log(result)
-  } catch (error) {
-    dispatch(contactFetchError(error));
+);
+
+export const deleteContact = createAsyncThunk(
+  actionTitles.deleteContact,
+  async (id) => {
+    const result = await axios.delete(`${MOCKAPI_URL}/contacts/${id}`);
+    if (result.status === 200) {
+      const result = await axios.get(`${MOCKAPI_URL}/contacts`);
+      return result.data;
+    } 
```

Comment:
1.  Please move `contacts to URL` and rename it
2. Please move api code to the separete file
3. We have `getContacts` already. Please use it here